news/feature-4137.md

```diff
@@ -0,0 +1,14 @@
+`http()`: Added compression ability for use with metered egress/ingress
+
+The new features can be accessed with the 
+ * `accept-encoding()` for inward http messages (currently not used by syslog-ng). The available options are `identity` (for no compression), `gzip` or `deflate` between quotation marks. If you want the driver to accept multiple compression types, you can list them separated by commas, or write `all`, if you want to enable all available compression types.
```

Comment:
Is this really not used by `syslog-ng`?
If yes, then why do we introduce a feature which does not get used by it?

It cannot be tested, it might break accidentally in the future, and confuses the users with the option.

Reply:
accept-encoding() reduces network traffic by making the server compress its response.
It has been tested, we just don't parse/use the response itself currently.

Reply:
Oh, okay, thanks for the clarification. It might be me, but I did not understand this sentence as you explained. Maybe we could rephrase this a bit, but it is optional.

Reply:
@alltilla do you have any suggestions on a clearer phrasing? I can't think of any better right now, but if you have suggestions, I'm open to ideas.

Reply:
What do you think about:

* `accept-encoding()` for requesting HTTP response compression from the server. ...

Reply:
I'll reword it a little bit, but it's a bit clearer tan my original.

Reply:
If I'm here, I'll rephrase some other things to be in a tad bit more correct English.

Reply:
Done